test/SourceKit/CodeComplete/complete_checkdeps_bridged.swift

```diff
@@ -36,6 +35,7 @@ func foo() {
 // RUN:   -req=complete -pos=5:3 %s -- ${COMPILER_ARGS[@]} == \
 
 // RUN:   -shell -- echo '### Modify bridging header library file' == \
+// RUN:   -shell -- sleep $SLEEP_TIME == \
 // RUN:   -shell -- cp -R $INPUT_DIR/MyProject_mod/LocalCFunc.h %t/MyProject/ == \
 // RUN:   -shell -- sleep $SLEEP_TIME == \
```

Comment:
Is the second sleep really needed?

Reply:
I believe it's not needed, but just I wanted to be on the safe side.

Reply:
Could we try removing it, it's better if we don't make tests unnecessarily slower.